tools/clang/test/HLSLFileCheck/hlsl/control_flow/attributes/unroll/unroll_switch_exit_crash.ll

```diff
@@ -0,0 +1,61 @@
+; RUN: %opt %s -dxil-loop-unroll,StructurizeLoopExits=1 -S | FileCheck %s
```

Comment:
This passes for me without crashing with an unaltered compiler and without the StructurizeLoopExits parameter. Is that expected? From the name of the file, I got the impression that this was going to crash

Reply:
Yes. The crash happens in the structurize code path, which is enabled in dxc.exe through `-opt-enable structurize-loop-exits-for-urnoll` (on by default), but disabled by default on dxopt since there was no way to enable that path without the new parameter.